src/Core/Console/AutomaticInstallation.php

```diff
@@ -20,19 +21,51 @@ protected function getHelp()
 		return <<<HELP
 Installation - Install Friendica automatically
 Synopsis
-	bin/console autoinstall [-h|--help|-?] [-v] [-a]
+	bin/console autoinstall [-h|--help|-?] [-v] [-a] [-f]
 
 Description
-    Installs Friendica with data based on the htconfig.php file
+    Installs Friendica with data based on the local.ini.php file or environment variables
 
-Notes:
+Notes
     Not checking .htaccess/URL-Rewrite during CLI installation.
 
 Options
     -h|--help|-? Show help information
     -v           Show more debug information.
     -a           All setup checks are required (except .htaccess)
-    -f           prepared config file (e.g. ".htconfig.php" itself)
+    -f|--file    prepared config file (e.g. "config/local.ini.php" itself) which will override every other config option - except the environment variables)
```

Comment:
Can you please differentiate the options that accept an additional value and those that don't ? Something like `-f|--file FILE`?